src/pages/explorer/auction.tsx

```diff
@@ -136,11 +138,19 @@ function PaginatedAuctionBids({
     const currentBids = (auction?.bidsByAuctionAddressAndAuctionName?.nodes ||
       []) as Bid[];
     const pageCount = Math.ceil(lastTotalBids / bidsPerPage);
+
     // Invoke when user click to request another page.
     const handlePageClick = (event) => {
       const newOffset = (event.selected * bidsPerPage) % lastTotalBids;
       setItemOffset(newOffset);
     };
+
+    const handleSortBy = (order) => {
+      const newSortOrder: BidsOrderBy = order;
+      setOrderBy(newSortOrder);
+      console.log(order)
+    }
```

Comment:
can we do something like this? or maybe in an arrow function since we're just calling 1 line of code? 

also as a general rule should keep `console.logs` out of the main branch
```suggestion
    const handleSortBy = (order: BidsOrderBy) => {
      setOrderBy(order);
    }
```